package.json

```diff
@@ -62,66 +62,6 @@
       "zone.js": "npm:zone.js@^0.6.17"
     },
     "overrides": {
-      "npm:@angular/common@2.0.0-rc.6": {
-        "meta": {
-          "src/*.js": {
-            "deps": [
-              "reflect-metadata",
-              "zone.js"
-            ]
-          }
-        }
-      },
-      "npm:@angular/compiler@2.0.0-rc.6": {
-        "meta": {
-          "src/*.js": {
-            "deps": [
-              "reflect-metadata",
-              "zone.js"
-            ]
-          }
-        }
-      },
-      "npm:@angular/core@2.0.0-rc.6": {
-        "meta": {
-          "src/*.js": {
-            "deps": [
-              "reflect-metadata",
-              "zone.js"
-            ]
-          }
-        }
-      },
-      "npm:@angular/http@2.0.0-rc.6": {
-        "meta": {
-          "src/*.js": {
-            "deps": [
-              "reflect-metadata",
-              "zone.js"
-            ]
-          }
-        }
-      },
-      "npm:@angular/platform-browser-dynamic@2.0.0-rc.6": {
-        "meta": {
-          "src/*.js": {
-            "deps": [
-              "reflect-metadata",
-              "zone.js"
-            ]
-          }
-        }
-      },
-      "npm:@angular/platform-browser@2.0.0-rc.6": {
-        "meta": {
-          "src/*.js": {
-            "deps": [
-              "reflect-metadata",
-              "zone.js"
-            ]
-          }
-        }
-      },
```

Comment:
Hm... are you sure these are not necessary? When running `npm run dev-install`, I've seen that these entries are added again.